metrics_utility/automation_controller_billing/collectors.py

```diff
@@ -102,21 +102,19 @@ def get_install_type():
 
 @register('config', '1.0', description=_('General platform configuration.'), config=True)
 def config(since, **kwargs):
-    license_info = get_license()
+    license_info, settings_info = get_config_and_settings_from_db()
     return {
         'platform': {
             'system': platform.system(),
             'dist': distro.linux_distribution(),
             'release': platform.release(),
             'type': get_install_type(),
         },
-        'install_uuid': settings.INSTALL_UUID,
-        'instance_uuid': settings.SYSTEM_UUID,
-        'controller_url_base': settings.TOWER_URL_BASE,
-        'controller_version': get_awx_version(),
-        'license_type': license_info.get('license_type', 'UNLICENSED'),
-        'license_date': license_info.get('license_date'),
-        'subscription_name': license_info.get('subscription_name'),
+        'install_uuid': settings_info.get('install_uuid'),
+        'tower_url_base': settings_info.get('tower_url_base'),
+        'controller_version': settings_info.get('version'),
+        'license': license_info.get('license', 'UNLICENSED'),
+        'subscription_name': license_info.get('subscription_name', ''),
```

Comment:
```suggestion
        'install_uuid': settings_info.get('install_uuid'),
        'instance_uuid': settings_info.get('system_uuid'),
        'controller_url_base': settings_info.get('tower_url_base'),
        'controller_version': settings_info.get('version'),
        'license_type': license_info.get('license_type', 'UNLICENSED'),
        'license_date': license_info.get('license_date'),
        'subscription_name': license_info.get('subscription_name'),
```

(instance_uuid was missing, controller_url_base was accidentally renamed, license_type & license_date became license, fixing to match the original output)